source/_integrations/alarm_control_panel.mqtt.markdown

```diff
@@ -108,6 +108,10 @@ command_topic:
   description: The MQTT topic to publish commands to change the alarm state.
   required: true
   type: string
+default_entity_id:
+  description: Use `default_entity_id` instead of name for automatic generation of the entity id. E.g. `alarm_control_panel.foobar`. When used without a `unique_id`, the entity id will update during restart or reload if the entity id is available.  If the entity id already exists, the entity id will be created with a number at the end. When used with a `unique_id`, the `default_entity_id` is only used when the entity is added for the first time. When set, this overrides a user-customized Entity ID in case the entity was deleted and added again.
```

Comment:
```suggestion
  description: Use `default_entity_id` instead of name for automatic generation of the entity ID. For example, `alarm_control_panel.foobar`. When used without a `unique_id`, the entity ID will update during restart or reload if the entity ID is available.  If the entity ID already exists, the entity ID will be created with a number at the end. When used with a `unique_id`, the `default_entity_id` is only used when the entity is added for the first time. When set, this overrides a user-customized entity ID if the entity was deleted and added again.
```
I know this is not currently consistent throughout the documentation, but I think [ID should be capitalized](https://www.merriam-webster.com/dictionary/id) in this context.

Reply:
if possible, [e.g. should be spelled out](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg): For example, such as, like, ..

Reply:
Replaced all instances based on your suggestion